### PR TITLE
Added personal userspace and keymaps.

### PR DESCRIPTION
**Pull request by SpacebarRacecar**
Adding my userspace and keymaps for the niu mini, planck and fc660c. I think the keymaps might be useful to some other german users. They emulate the US International layout on PCs that have German set as input language. That means that the keyboard can be plugged into any PC in Germany, even public ones where the user doesn't have permission to change the settings, and still function as a US International layout.

### CODE REVIEW

File: keyboards/fc660c/keymaps/spacebarracecar/config.h [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/fc660c/keymaps/spacebarracecar/config.h]: This isn't actually needed, as the `config_common.h` file should be included in the keyboard's config.h.

File: keyboards/niu_mini/keymaps/spacebarracecar/keymap.c [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  _______,  KC_LEFT,  KC_DOWN,  KC_RGHT,  KC_END,   XXXXXXX,  XXXXXXX,  GUIL,     GUID,     GUIR,     RGB_M_P,  KC_ENT,
+  _______,  KC_MPRV,  KC_MPLY,  KC_MNXT,  KC_VOLD,  KC_VOLU,  KC_MUTE,  RGB_TOG,  RGB_MOD,  RGB_HUI,  CU_RGBV,  _______,
+  RESET,    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  CU_GAME
+)
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+switch (keycode) {
+  case MO(LOWER):
```
Comment (drashna) [inline keyboards/niu_mini/keymaps/spacebarracecar/keymap.c]: Shouldn't this be `MO(_LOWER)`?

**Comment by SpacebarRacecar** [comment]
I carried out the requested changes and also did some minor changes to my userspace.

**Comment by drashna** [comment]
Nice, thanks!

**Pull request merged**